dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -830,3 +837,50 @@ CWWKD1082.entity.classes.missing.explanation=Repositories use entity classes \
  that must be defined in the persistence unit.
 CWWKD1082.entity.classes.missing.useraction=Include all entity classes \
  in the persistence unit.
+
+CWWKD1083.dup.method.param=CWWKD1083E: Multiple parameters of the {0} method \
+ of the {1} repository interface correspond to the named parameter: {2}. \
+ You can annotate one method parameter with {3} to specify the query language \
+ named parameter. Alternatively, you can enable the -parameters compile option \
+ and define one method parameter of the form, {4}, which does not require the \
+ Param annotation.
+CWWKD1083.dup.method.param.explanation=Each method parameter must correspond \
+ to exactly one query parameter.
+CWWKD1083.dup.method.param.useraction=Check if each of the method parameters \
+ specifies the correct named parameter. Remove method parameters that are \
+ duplicates.
+
+CWWKD1084.missing.named.params=CWWKD1084E: The parameters of the {0} method \
+ of the {1} repository interface do not define the named parameters ({2}) \
+ that are required by the Query annotation value: {3}. You can define a \
+ named parameter for the query by placing the Param annotation on a \
+ corresponding method parameter. For example, {4}. Alternatively, if you \
+ enable the -parameters compile option, you can omit the Param annotation \
+ if you name the method parameter to have the same name as the named parameter. \
+ For example, if String is the named parameter type, the method parameter \
+ can be {5}, which does not require the Param annotation.
+CWWKD1084.missing.named.params.explanation=The Query annotation value requires \
+ one or more named parameters that are not supplied by the repository method.
+CWWKD1084.missing.named.params.useraction=Add parameters to the repository method \
+ to specify the missing named parameters.
+
+CWWKD1085.extra.method.params=CWWKD1085E: The parameters of the {0} method \
+ of the {1} repository interface define query parameters ({2}) that do not \
```

Comment:
```suggestion
 in the {1} repository interface define query parameters ({2}) that do not \
```

Reply:
The phrase "method of the {#} repository" already appears more than 50 times within this same message file, which is all new messages this release that were reviewed by the ID team over the last couple of months, so I'm not going to change this one occurrence to use "in" instead of "of".  I'm fine with the change from "that" to ", which". I'll cover it in the commit where I am applying your other changes.